client-tools/Benchmark/BenchFeature.cpp

```diff
@@ -250,6 +257,18 @@ void BenchFeature::validateOptions(std::shared_ptr<ProgramOptions> options) {
           << _percentiles << "': histogram is disabled by default. Enable it with flag '--histogram.generate = true'.";
     }
   }
+  LOG_DEVEL << _customQuery;
+  if (!_customQueryBindVars.empty()) {
+    _customQueryBindVarsBuilder = VPackParser::fromJson(_customQueryBindVars);
+  }
+
+  if(!options->processingResult().touched("--custom-query") && !options->processingResult().touched("--custom-query-file")) {
```

Comment:
I think we can get away without this warning.

Reply:
Addressed